client/src/components/DetailsEvent/index.js

```diff
@@ -1,13 +1,180 @@
+                  <label className="title">Organization name:</label>
+                  <label className="answer">{org_name}</label>
+                </div>
+                <div className="labels_container">
+                  <label className="title">Hall cost:</label>
+                  <label className="answer">{price}</label>
+                </div>
+                <div className="labels_container">
+                  <label className="title">Number of persons:</label>
+                  <label className="answer">{capacity}</label>
+                </div>
+                <div className="labels_container">
+                  <label className="title">Start Date:</label>
+                  <label className="answer">{start_ddmmyyyy}</label>
+                </div>
+                <div className="labels_container">
+                  <label className="title">End Date:</label>
+                  <label className="answer">{end_ddmmyyyy}</label>
+                </div>
+                <div className="labels_container">
+                  <label className="title">note:</label>
+                  <label className="answer">{note}</label>
+                </div>
+                <h2 className="h2">Lunch</h2>
+                <div className="labels_container">
+                  <label className="title">Description:</label>
+                  <label className="answer">{lunch_note}</label>
+                </div>
+                <div className="labels_container">
+                  <label className="title">time:</label>
+                  <label className="answer">{lunch_time}</label>
+                </div>
+                <div className="labels_container">
+                  <label className="title">price per person:</label>
+                  <label className="answer">{lunch_price}</label>
+                </div>
+                <h2 className="h2">Coffee</h2>
+                <div className="labels_container">
+                  <label className="title">Description:</label>
+                  <label className="answer">{coffee_note}</label>
+                </div>
+                <div className="labels_container">
+                  <label className="title">time:</label>
+                  <label className="answer">{coffee_time}</label>
+                </div>
+                <div className="labels_container">
+                  <label className="title">price per person:</label>
+                  <label className="answer">{coffee_price}</label>
+                </div>
+                <h2 className="h2">Equipment</h2>
+                <div className="labels_container">
+                  <label className="title">Description:</label>
+                  <label className="answer">{equipment_note}</label>
+                </div>
+                <div className="labels_container">
+                  <label className="title">price:</label>
+                  <label className="answer">{equipment_price}</label>
```

Comment:
This is all _very_ repetitive, can you find a way to loop over an array to do this?